pkg/ingress/controller/apisix_service.go

```diff
@@ -89,16 +95,15 @@ func (c *ApisixServiceController) processNextWorkItem() bool {
 	}
 	err := func(obj interface{}) error {
 		defer c.workqueue.Done(obj)
-		var key string
+		var sqo *ServiceQueueObj
 		var ok bool
-
-		if key, ok = obj.(string); !ok {
+		if sqo, ok = obj.(*ServiceQueueObj); !ok {
 			c.workqueue.Forget(obj)
 			return fmt.Errorf("expected string in workqueue but got %#v", obj)
```

Comment:
Typo, should be "expected ServiceQueueObj object in workqueue but got %#v".

Reply:
Done